src/components/select-controls/radial/Radial.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import { colors } from "../../colors/index";
+
+const Radial = (props) => {
+
+    const [checked, setChecked] = useState(props.checked)
+
+    const Radial = styled.div`
+        height: 23px;
+        width: 23px;
+        border: 3px solid;
+        border-radius: 100%;
+        cursor: ${p=>p.active ? 'pointer' : 'default'};
+        background: transparent;
+        position: relative;
+        box-shadow: 0 2px 5px rgba(0,0,0,0.2);
+        ${p=>p.active 
+            ? `border-color: ${colors.emerald400};` 
+            : `border-color: ${colors.grey600};`}
+
```

Comment:
The radial component is supposed to be white when active and emerald when hovered upon. You can do this by removing lines 19-21 and instead adding `${(p) => getBorderColor(p)}` and  
```
&:hover {
      border-color: ${colors.emerald400};
    } 
``` 

  const getBorderColor = (p) => {
    if (!p.active) {
      return `border-color: ${colors.grey600};`;
    }
    if (checked) return `border-color: ${colors.emerald400};`;
    return `border-color: white;`;
  };

Reply:
TY! 😍

Reply:
@naijwu any update here?